CRM/Contact/BAO/ContactType.php

```diff
@@ -385,8 +385,9 @@ public  static function getSelectElements(
     }
 
     $argString = $all ? 'CRM_CT_GSE_1' : 'CRM_CT_GSE_0';
-    $argString .= $isSeparator ? '_1' : '_0';
+    $argString .= $isSeparator ? '_1' : ' _0';
```

Comment:
why add a space here?

Reply:
Nice catch! Don't think that should be there. I'll squash it.